test/common/README.md

```diff
@@ -261,6 +261,124 @@ Platform normalizes the `pwd` command.
 
 Synchronous version of `spawnPwd`.
 
```

Comment:
However, this will break TOC structure and API reference alphabet principle. So I would abstain.